app/celery/tasks.py

```diff
@@ -596,7 +596,7 @@ def process_incomplete_job(job_id):
     job_complete(job, resumed=True)
 
 
-def queue_to_use(notifications_count: int) -> str:
+def queue_to_use(notifications_count: int):
```

Comment:
Same thing here... were these type annotations wrong ?

Reply:
This function could return `None` so the `str` was not the correct return type. I will change this so the return type is `Optional[str]`.